invenio_communities/views/communities.py

```diff
@@ -216,6 +216,22 @@ def communities_settings_privileges(pid_value, community, community_ui):
     )
 
 
+@pass_community(serialize=True)
+def communities_settings_curation_policy(pid_value, community, community_ui):
+    """Community settings/curation-policy page."""
+    permissions = community.has_permissions_to(
+        ["update", "read", "search_requests", "search_invites"]
+    )
+    if not permissions["can_update"]:
+        raise PermissionDeniedError()
+
+    return render_template(
+        "invenio_communities/details/settings/curation_policy.html",
+        community=community_ui,
+        permissions=permissions,
```

Comment:
Minor: Do we need to pass the permissions? I've checked and looks like it's not really used (unless I am mistaken).

Same comment for the functions below.

Reply:
They are used in the submenu in the header: 
https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/templates/semantic-ui/invenio_communities/details/header.html#L133